src/main/java/com/exasol/repository/ReleaseChangesLetter.java

```diff
@@ -0,0 +1,164 @@
+package com.exasol.repository;
+
+import static com.exasol.ReleaseRobotConstants.*;
+
+import java.time.LocalDate;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * This class represents a changes file's content.
+ */
+public class ReleaseChangesLetter {
+    private final String fileName;
+    private String header = null;
+    private String body = null;
+    private String versionNumber = null;
+    private LocalDate releaseDate = null;
+    private final List<Integer> ticketNumbers = new ArrayList<>();
+
+    /**
+     * Create a new {@link ReleaseChangesLetter} from a string.
+     * 
+     * @param fileName name of the changes file
+     * @param content file's content as a string
+     */
+    public ReleaseChangesLetter(final String fileName, final String content) {
+        this.fileName = fileName;
+        if (content != null && content.length() > 1) {
+            parseContent(content);
+        }
+    }
+
+    /**
+     * Get a name of the changes file.
+     * 
+     * @return name of the changes file
+     */
+    public String getFileName() {
+        return this.fileName;
+    }
+
+    /**
+     * Get a version number.
+     * 
+     * @return version number as a string
+     */
+    public Optional<String> getVersionNumber() {
+        return this.versionNumber == null ? Optional.empty() : Optional.of(this.versionNumber);
```

Comment:
```suggestion
        return Optional.ofNullable(this.versionNumber);
```

If you have Nullable content that can be queried via get, it is a lot more elegant to split that up into a `boolean hasVersionNumber()` method and a `getVersionNumber()` method which can then safely return null instead of wrapping the value into an `Optional`.

Reply:
I think it's not really safe to return nulls and I prefer to use optional. You can forget to check if it's null or not even if you have that convenient method 'hasVersionNumber'. And I can't forget to check it if I get Optional. Even if's not so elegant, it's safe